src/OrbitQt/orbitmainwindow.h

```diff
@@ -58,10 +58,12 @@ class OrbitMainWindow final : public QMainWindow, public orbit_gl::MainWindowInt
   static constexpr int kQuitOrbitReturnCode = 0;
   static constexpr int kEndSessionReturnCode = 1;
 
-  explicit OrbitMainWindow(orbit_session_setup::TargetConfiguration target_configuration,
-                           const orbit_base::CrashHandler* crash_handler,
-                           orbit_metrics_uploader::MetricsUploader* metrics_uploader = nullptr,
-                           const QStringList& command_line_flags = QStringList());
+  explicit OrbitMainWindow(
```

Comment:
I would rather not expose the BinomialConfidenceInterfacelEstimator to outside of OrbitApp, in particular orbitmainwindow and main. In particular I would not add the estimator to this constructor here.

The estimator to choose from is really an implementation detail, and we should not worry about it in the UI parts (at least as long as the user has no chance to choose between different estimators).

Instead, I would simply let OrbitApp instantiate the estimator by itself.

Reply:
Davide suggests the same thing. So I copy-paste my reasoning

> Hypothetically, in the future we might want to add a command line argument controlling the confidence level (e.g. 95% vs 99%) or turning the thing off altogether.

Reply:
OK. Davide's convinced me.

Reply:
Thanks